src/mafiasi_link_shortener/links/views.py

```diff
@@ -1,12 +1,18 @@
 from django.core.exceptions import ObjectDoesNotExist
 from django.http import Http404, HttpResponse, HttpResponseRedirect
+from rest_framework.decorators import api_view, permission_classes
 
 from . import models
 
 
+@api_view(["GET"])
+# no permissions required because we check for them depending on the link
+@permission_classes([])
```

Comment:
Isn't this a native django view? In that case the rest_framework permissions aren't checked anyways.

Reply:
Alright yeah, i overlooked that